notebooks/Notebook1.md

```diff
@@ -18,17 +20,29 @@ exciting milestones, preliminary results, etc. Feel free to include images
 code, and any other resources that you think will help clearly convey your
 design process.
 
+One of my big decisions this week was choosing a host language. The three main languages I was considering were scala, R, and Python. I liked the idea for using R because it has a lot of built in things for data and data visualization that would save time, however I also liked the idea of using Python becuase it is a much simpler language than R and still does have good data visualization. The ease of use for the programmer is something really inportant to me, so I decided to not use R. I decided to use Scala as the host language because I have really enjoyed using it so far in this class and wanted to work with it more. I also found that it had lots of data visualization librariesan use to make my graphs. I found this website: https://analyticsindiamag.com/top-7-scala-libraries-for-data-visualisation/ that listed out a bunch of the top ibraries. I think I am going to use EvilPlot mainly because I just like the name. https://github.com/cibotech/evilplot It has teh capability to do line graphs which is all I need for this project. 
+
+Now I want to talk about specific graphs that you would make with my language. All the graphs mainly be would be line graphs. You could use it to track change in weight over time and compare that to amount of calories eaten on each day or the amount of calories burned each day. You could also make a pie chart of the the percentage of each type of food eaten in a given period of time. 
+
 ## Questions
 
 **What is the most pressing issue for your project? What design decision do
 you need to make, what implementation issue are you trying to solve, or how
 are you evaluating your design and implementation?**
 
+The most pressuring issue was deciding how  was going to display the graphs. That one decision stemmed for a lot of other design questions like what host language to choose and what stats to track. 
+
 **What questions do you have for your critique partners? How can they best help
 you?**
 
+What other graphs would you want to make from this data? Would you find this langiage useful? 
+
 **How much time did you spend on the project this week? If you're working in a
 team, how did you share the work?**
 
+I have only spent a few hours outside of class this week, but I plan on mainly working on it tomorrow and tuesday.
+
 **Compared to what you wrote in your contract about what you want to get out of this
 project, how did this week go?**
+
+This week is going great. I have made most of my main decisions and just have to finalize some decisions on which data to track. 
```

Comment:
Great! I think Scala is a good choice if you are comfortable with it, and it has the plotting features that you need.